command/c/chown.md

```diff
@@ -0,0 +1,282 @@
+# `chown`
+
+## Description
+
+`chown` stands for change owner which changes user or group ownership of a file, directory, or symbolic link. Every file or directory has group or user ownership in Linux. Since multiple users and groups can use same linux system so they all have unique names and IDs. Name and ID of a user and a group can be same. 
```

Comment:
```suggestion
`chown` stands for change owner which changes user or group ownership of a file, directory, or symbolic link. Every file or directory has group or user ownership in Linux. Since, multiple users and groups can use same Linux system so they all have unique names and IDs. Name and ID of a user and a group can be same. 
```